cfme/services/catalogs/catalog_items/ansible_catalog_items.py

```diff
@@ -160,7 +160,13 @@ class retirement(WaitTab):  # noqa
 class AddAnsibleCatalogItemView(AnsibleCatalogItemForm):
     add = Button("Add")
 
-    is_displayed = displayed_not_implemented
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and
+            self.title.text == 'Adding a new Service Catalog Item' and
+            self.provisioning.repository  # this field should be visible
```

Comment:
Is .is_displayed getting called here implicitly in the `__get__`  of repository? `Widget.is_displayed` should resolve for any widget that doesn't define its own is_displayed, and this should be safe to call in this case if the widget has its locator passed correctly.